### PR TITLE
security: Add a security overview

### PR DESCRIPTION
**Pull request by doanac**
Signed-off-by: Andy Doan <andy@foundries.io>

### CODE REVIEW

File: source/reference-manual/security/security.rst [hunk 1]
```diff
@@ -10,3 +10,70 @@ Security
...
+   TUF key management for signing what can be installed. It also
+   includes how firmware is signed so that bootloaders will trust it.
+
+ * Device security - how devices store secure artifacts.
+
+ * Connection security - how devices and cloud services trust each
+   other.
+
+TUF Security
+------------
```
Comment (ricardosalveti) [inline source/reference-manual/security/security.rst:30]: Maybe also point out to https://theupdateframework.io/security/ (and suggest going over the specification?)

**Review by ricardosalveti — APPROVED** [review]
LGTM, I can extend the secure boot piece once this is merged.

File: source/reference-manual/security/security.rst [hunk 2]
```diff
@@ -10,3 +10,70 @@ Security
...
+
+.. _provisioning process:
+   https://docs.foundries.io/latest/reference-manual/security/factory-registration-ref.html
+
+Connection Security
+-------------------
+
+Connection security ties everything together. Devices and the `device
+gateway`_ use mutual TLS to establish trust. We include a stream
+lined way to establish PKI_ for each factory that works with the
```
Comment (kprosise) [inline source/reference-manual/security/security.rst:72]:
```suggestion
gateway`_ use mutual TLS to establish trust. We include a
streamlined way to establish PKI_ for each factory that works with the
```

**Review by kprosise — APPROVED** [review]
Just some minor suggestions, otherwise it LGTM.

**Comment by doanac** [comment]
Docs for 93cc3789f336db05f53bf26f5d21a5b5f8ae7dd3 are browesable at: https://ci.foundries.io/projects/fio-docs/builds/1271/docs/artifacts/html/index.html

**Comment by doanac** [comment]
Docs for 2c7bda2a185d7416a868bac05838c8de3a43841c are browesable at: https://ci.foundries.io/projects/fio-docs/builds/1273/docs/artifacts/html/index.html

**Comment by doanac** [comment]
Docs for 2a57bfbc9f90964de4a754b8ac86867756fb3995 are browesable at: https://ci.foundries.io/projects/fio-docs/builds/1272/docs/artifacts/html/index.html

**Comment by doanac** [comment]
Docs for 2c7bda2a185d7416a868bac05838c8de3a43841c are browesable at: https://ci.foundries.io/projects/fio-docs/builds/1274/docs/artifacts/html/index.html

File: source/reference-manual/security/security.rst (hunk 1 above)
Comment (doanac) [reply source/reference-manual/security/security.rst:30]: https://github.com/foundriesio/docs/pull/316/commits/00531d337c42fd9b46c2ed7831a3d63e757c987b

**Comment by doanac** [comment]
Docs for 00531d337c42fd9b46c2ed7831a3d63e757c987b are browesable at: https://ci.foundries.io/projects/fio-docs/builds/1275/docs/artifacts/html/index.html

**Comment by doanac** [comment]
NOTE: there are some bad links in this - i'll fix them in another PR

**Pull request merged**